class.jetpack.php

```diff
@@ -964,8 +964,14 @@ public static function show_development_mode_notice() {
 				$notice = __( 'In Development Mode, via the jetpack_development_mode filter.', 'jetpack' );
 			}
 
-			$output = '<div class="error"><p>' . $notice . '</p></div>';
-			echo $output;
+			echo '<div class="error"><p>' . $notice . '</p></div>';
+		}
+
+		// Throw up a notice if using a development version and as for feedback.
+		if ( Jetpack::is_development_version() ) {
+			$notice = sprintf( __( 'You are currently running a development version of Jetpack.  %1sSubmit your feedback%2s', 'jetpack' ), '<a href="https://jetpack.me/contact-support/beta-group/" target="_blank">', '</a>' );
```

Comment:
I have a couple of suggestions for this string. First of all, let's add a period to the end of the sentence that will be the link, and second - let's separate HTML tag placeholders from the text with spaces. Browsers don't care about spaces, and it will make translators' lives a little easier. Also, we should probably use `_x()` and explain that `%1s` and `%2s` are just HTML tags and will not add any words to the string.